pcdsdevices/ui/SmarActTipTilt.embedded.py

```diff
@@ -119,6 +121,60 @@ def set_open_loop(self, axis: str):
         set_open_loop(self, axis='tip')
         set_open_loop(self, axis='tilt')
 
+        self.ui.invert_tip.stateChanged.connect(self._invert_tip_axis)
+        self.ui.invert_tilt.stateChanged.connect(self._invert_tilt_axis)
+
+    def invert_axis(self, axis: str, invert: bool = False):
+        """
+        Invert the jog pushbuttons
+
+        Parameters
+        -----------
+            axis: str
+                Name of axis
+            invert: bool
+                Whether or not to invert. Default is False.
```

Comment:
```suggestion
        axis: str
            Name of axis
        invert: bool
            Whether or not to invert. Default is False.
```
Giga-nit: I think these are supposed to be de-dented one level

Reply:
Valid, I'll gladly accept